go/test/endtoend/cluster/vtorc_process.go

```diff
@@ -51,10 +51,6 @@ type VTOrcProcess struct {
 type VTOrcConfiguration struct {
 	Debug                                 bool
 	ListenAddress                         string
-	MySQLTopologyUser                     string
-	MySQLTopologyPassword                 string
-	MySQLReplicaUser                      string
-	MySQLReplicaPassword                  string
```

Comment:
@GuptaManan100 Should these also be removed from `config/vtorc/default.json, `examples/common/vtorc/config.json`, `examples/compose/vtorc-up.sh` and `examples/compose/vtorc/default.json`?

Reply:
Done!